account_test.go

```diff
@@ -0,0 +1,218 @@
+package dcrlibwallet_test
+
+import (
+	"os"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/raedahgroup/dcrlibwallet"
+)
+
+const rootDir = ".dcrlibwallet_tests_root"
+
+var _ = Describe("Account", func() {
+	var (
+		wallet *Wallet
+	)
+
+	BeforeEach(func() {
+		multi, err := NewMultiWallet(rootDir, "", "testnet3")
+		Expect(err).To(BeNil())
+		Expect(multi).ToNot(BeNil())
+
+		wallet, err = multi.CreateNewWallet("0000", 0)
```

Comment:
Randomizing the password and passtype seems like a good idea. (it's not worth the stress if it takes anymore dev/running time)